src/portal/src/app/project/project-detail/project-detail.component.html

```diff
@@ -1,8 +1,14 @@
 <a *ngIf="hasSignedIn" (click)="backToProject()" class="backStyle"> {{'PROJECT_DETAIL.PROJECTS' | translate}}</a>
 <a *ngIf="!hasSignedIn" [routerLink]="['/harbor', 'sign-in']"> {{'SEARCH.BACK' | translate}}</a>
 
-<h1 class="custom-h2" sub-header-title>{{currentProject.name}} <span class="role-label"
-        *ngIf="isMember">{{roleName | translate}}</span></h1>
+<h1 class="custom-h2" sub-header-title>
+    <clr-icon *ngIf="isProxyCacheProject" shape="cloud-traffic" size="36"></clr-icon>
```

Comment:
The picture is a bit big compared to the design

Reply:
fixed